src/Microsoft.OpenApi/Services/OpenApiWalker.cs

```diff
@@ -470,10 +470,32 @@ internal void Walk(OpenApiOperation operation)
             Walk(OpenApiConstants.Responses, () => Walk(operation.Responses));
             Walk(OpenApiConstants.Callbacks, () => Walk(operation.Callbacks));
             Walk(OpenApiConstants.Tags, () => Walk(operation.Tags));
-
+            Walk(OpenApiConstants.Security, () => Walk(operation.Security));
```

Comment:
If we miss add the process code, it should not be failed. So, I can't understand why? Is there any default setting that we can fix it?

Reply:
I'm not sure I follow.  If we are missing Walk() methods then there may be references that remain unresolved or objects not validated.  Technically that's not an error as Unresolved references is a valid state.  They could be resolved manually later.

Yes, there has been an unfortunate number of cases of missing resolved references.  In theory these should have been picked up by regular test cases as unresolved references have null/default property values..  Not quite sure why more were not.